src/main/java/org/openmaptiles/layers/Poi.java

```diff
@@ -156,6 +156,18 @@ private <T extends Tables.WithSubclass & Tables.WithStation & Tables.WithFunicul
       }
     }
 
+    // Parcel locker without name: use either brand or operator and add ref if present
+    if ("parcel_locker".equals(rawSubclass) && nullOrEmpty(name)) {
+      name = coalesce(nullIfEmpty(element.brand()), nullIfEmpty(element.operator()));
+      String ref = nullIfEmpty(element.ref());
+      if (ref != null) {
+        name = name == null ? ref : name + " " + ref;
```

Comment:
might want to group the conditional to make it clear that it's `name == null ? ref : (name + " " + ref)` and not `(name == null ? ref : name) + " " + ref`

Reply:
adjusted